pkg/target/target.go

```diff
@@ -337,12 +344,13 @@ func (h *K8sValidationTarget) MatchSchema() apiextensions.JSONSchemaProps {
 					},
 				},
 			},
-			"namespaces":         wildcardNSList,
-			"excludedNamespaces": wildcardNSList,
-			"labelSelector":      labelSelectorSchema,
-			"namespaceSelector":  labelSelectorSchema,
+			"namespaces":         propsWithDescription(wildcardNSList, "namespaces is a list of namespace names. If defined, a constraint will only apply to resources in a listed namespace."),
+			"excludedNamespaces": propsWithDescription(wildcardNSList, "excludedNamespaces is a list of namespace names. If defined, a constraint will only apply to resources not in a listed namespace."),
+			"labelSelector":      propsWithDescription(labelSelectorSchema, "labelSelector is a standard Kubernetes label selector."),
+			"namespaceSelector":  propsWithDescription(labelSelectorSchema, "namespaceSelector is a standard Kubernetes namespace selector. If defined, make sure to add Namespaces to your configs.config.gatekeeper.sh object to ensure namespaces are synced into OPA."),
 			"scope": {
-				Type: "string",
+				Type:        "string",
+				Description: "scope accepts *, Cluster, or Namespaced which determines if cluster-scoped and/or namesapced-scoped resources are selected. (defaults to *)",
```

Comment:
Wrap values in backticks. Also s/selected/matched/

Reply:
Done